### ISSUE TITLE
Google Docs

### ISSUE THREAD

**Issue by SteegHobland** [ssborbis/ContextSearch-web-ext#739]
Is there any way to get this extension to work on Google Docs? Edit to add: By using the context menu, I mean. Not just by using the toolbar button.

**Comment by ssborbis** [comment]
I don't think so. Google docs uses some fancy SVG magic to display and edit the documents. This addon works off your standard HTML elements. Even the browser's own context menu can't read the highlighted words. I'm not saying it's impossible, but it would require a LOT.

**Comment by SteegHobland** [comment]
Ah… I will not be getting my hopes up, then!
